src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+    private List<Integer> rangeNumbers = IntStream.rangeClosed(1, 45).boxed().collect(Collectors.toList());
+    private final List<Integer> lotto;
+
+
+    public Lotto() {
+        Collections.shuffle(rangeNumbers);
+        lotto = rangeNumbers.stream()
+                .limit(6)
+                .sorted()
+                .collect(Collectors.toList());
+    }
+
+    public Lotto(List<Integer> randomNumbers) {
+        lotto = randomNumbers;
+    }
+
+    public Rank winningResult(WinningNumber winningNumber) {
+        int matchCount = (int) winningNumber.getWinningNumbers()
+                .stream()
+                .filter(number -> this.lotto.contains(number))
```

Comment:
```suggestion
                .filter(this.lotto::contains)
```
이렇게 람다로 표현하면 인자를 적지 않거나 한번만 적어도 되어서 좀 더 깔끔하다고 생각해요. 🙂